cetmix_tower_tldextract/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2025 Cetmix OÜ
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Cetmix Tower TLDExtract",
+    "summary": """Provides wrapped tldextract for tower modules""",
+    "version": "14.0.1.0.0",
+    "category": "Productivity",
+    "license": "AGPL-3",
+    "author": "Cetmix",
```

Comment:
Keep all the [authors](https://github.com/cetmix/cetmix-tower/blob/19564e03dce457f3ddbf40fa809cf24968ab90db/cetmix_tower_ovh/__manifest__.py#L11C24-L11C38) and [maintainers](https://github.com/cetmix/cetmix-tower/blob/19564e03dce457f3ddbf40fa809cf24968ab90db/cetmix_tower_ovh/__manifest__.py#L12) of the original module.